src/mame/act/victor9k.cpp

```diff
@@ -738,6 +744,11 @@ void victor9k_state::victor9k(machine_config &config)
 	//MCFG_HC55516_DIG_OUT_CB(WRITELINE(MC6852_TAG, mc6852_device, rx_w))
 	m_cvsd->add_route(ALL_OUTPUTS, "mono", 0.25);
 
+	MC6852(config, m_ssda, 0);
+	m_ssda->tx_data_callback().set(HC55516_TAG, FUNC(hc55516_device::digit_w));
+	m_ssda->sm_dtr_callback().set(FUNC(victor9k_state::ssda_sm_dtr_w));
+	m_ssda->irq_callback().set(FUNC(victor9k_state::ssda_irq_w));
+
```

Comment:
Why was this moved?  It doesn’t seem to be changed.

Reply:
I was trying to pull the sound hardware together. The comments say sound related HW and goes to other I/O devices. While this is a serial chip it exists only to drive the sound chip. It seemed misleading to put it with the UART code.